TWLight/applications/views.py

```diff
@@ -474,6 +474,20 @@ def dispatch(self, request, *args, **kwargs):
             messages.add_message(request, messages.ERROR, message)
             return HttpResponseRedirect(url, message)
 
+        # Non-elligible users should redirect to my library page
+        editor = self.request.user.editor
+        is_editor_elligible = all(
```

Comment:
Eligibility is actually a little more complicated than this, which is why we have a helper that you can import:
`from TWLight.users.helpers.editor_data import editor_bundle_eligible`
It takes an editor object as the argument and returns true/false. This whole block of code should be able to be replaced with something like:
```
        if not editor_bundle_eligible(self.request.user.editor):
            url = reverse("users:my_library")
            return HttpResponseRedirect(url)
```